client/src/components/files/FileDropZone.vue

```diff
@@ -5,21 +5,52 @@
     class="drop-zone"
     :class="isActive ? 'drop-zone-active' : 'drop-zone-inactive'"
     @drop.prevent="onDrop"
-    @dragenter.prevent="isActive = true"
-    @dragleave.prevent="setInactive()"
+    @dragenter.prevent="onDragEnter()"
+    @dragleave.prevent="onDragLeave()"
   >
     <slot />
+
+    <div
+      v-show="isActive && props.showDropMessage"
+      class="drop-message"
+    >
+      <ms-image
+        :image="AddDocument"
+        class="restore-password-header-img"
+      />
+      <ion-label class="subtitles-normal">
+        {{ $msTranslate('FoldersPage.ImportFile.dropInstructions') }}
+      </ion-label>
+    </div>
   </div>
 </template>
 
 <script setup lang="ts">
-import { onMounted, onUnmounted, ref } from 'vue';
+import { AddDocument, MsImage } from '@/components/core';
+import { FileImportTuple, getFilesFromDrop } from '@/components/files/utils';
+import { FsPath } from '@/parsec';
+import { IonLabel } from '@ionic/vue';
+import { computed, onMounted, onUnmounted, ref } from 'vue';
+
+defineExpose({
+  reset,
+});
+
+const props = defineProps<{
+  currentPath: FsPath;
+  disabled?: boolean;
+  showDropMessage?: boolean;
```

Comment:
How useful is the `showDropMessage` prop ? It's set to `true` in all cases.

Reply:
Nop, not when dropping on a file/folder. When it's set, it's indeed set to true. But there are cases when it's not set.